packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -39,7 +39,7 @@ rm -rf lastpass-cli
 mkdir -p /home/ubuntu/.local/share/lpass
 
 # install puppeteer dependencies https://pptr.dev/15.3.0/troubleshooting#chrome-headless-doesnt-launch-on-unix
-sudo apt-get -y install \
+sudo apt-get -yqq install \
```

Comment:
dependencies didn't install in gold master, see if this will fix.

Reply:
Curious to know why adding `qq` to the args would fix it? Doesn't that just affect logging?